packages/main/src/plugin/image-layers.ts

```diff
@@ -0,0 +1,123 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import nodeTar from 'tar';
+import fs from 'node:fs';
+import path from 'node:path';
+import { FileTree } from './file-tree.js';
+
+export interface Colorable {
```

Comment:
probably need to be FileColorable as it is related to files

it will avoid name collision as it's exported